contrib/mesos/pkg/scheduler/components/framework/framework.go

```diff
@@ -128,14 +123,6 @@ func New(config Config) Framework {
 					return false
 				}
 
-				// the executor IDs must not identify a kubelet-executor with a group that doesn't match ours
```

Comment:
now that this code is deleted, how is this requirement met? we don't want to launch tasks for executor's that have, for example, different command line flags.